pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -579,11 +579,14 @@ func TestExpireAddUpdatePod(t *testing.T) {
 func TestRemovePod(t *testing.T) {
 	nodeName := "node"
 	basePod := makeBasePod(t, nodeName, "test", "100m", "500", "", []v1.ContainerPort{{HostIP: "127.0.0.1", HostPort: 80, Protocol: "TCP"}})
+	podWithEphemeralStorage := makePodWithEphemeralStorage(basePod, "100")
 	tests := []struct {
 		pod       *v1.Pod
+		ephPod    *v1.Pod
 		wNodeInfo *NodeInfo
 	}{{
-		pod: basePod,
+		pod:    basePod,
+		ephPod: podWithEphemeralStorage,
```

Comment:
I think you didn't need to change this `test`. Couldn't you just add another `test` similar to this one with ephemeral storage? I guess you wouldn't need to change anything inside the `for` loop in that case and everything would be simpler.

Reply:
yeah. at first i think they can do the same thing and reuse some similar code , So i add ephemeral storage stuff to this test case. anyway i can add another test.